src/superphot_plus/fit_numpyro.py

```diff
@@ -385,7 +385,7 @@ def run_mcmc_batch(lcs, t0_lim=None, plot=False):
         tdata_stacked.append(lc.times)
         fdata_stacked.append(lc.fluxes)
         ferrdata_stacked.append(lc.flux_errors)
-        bdata_stacked.append(np.where(lc.bands == "r", ref_band_idx, 1 - ref_band_idx))  # change to integers
+        bdata_stacked.append(lc.band_as_int(["r", "g"]))  # change to integers
```

Comment:
Should these be in the same order? e.g. "r/g" instead of "g/r"?

Reply:
Good catch!!!